pkgs/applications/networking/cluster/habitat/default.nix

```diff
@@ -1,24 +1,22 @@
 { lib, fetchFromGitHub, rustPlatform, pkgconfig
-, libsodium, libarchive, openssl }:
+, libsodium, libarchive, openssl, zeromq }:
 
 with rustPlatform;
 
 buildRustPackage rec {
   name = "habitat-${version}";
```

Comment:
Avoid repetition

```suggestion
   pname = "habitat";
```

Reply:
I’ve seen it either way here, so leaving it as it was unless someone else feels strongly about it.